### ISSUE TITLE
Display does not handle switching screen resolution

### ISSUE THREAD

**Issue by Ronsss** [rcmaehl/WhyNotWin11#569]
**Describe the bug**

I have a dual monitor setup, HD 1920x1080 laptop monitor and 4K 3840x2160 resolutions.

App launched on laptop monitor fine (maybe displaying a little "high", separate issue).

Moved it to the 4K display, still OK.

Move it back to the Laptop display. NOT OK, "squashed" image
![image](https://user-images.githubusercontent.com/33688532/130501286-376c685e-4f4b-4575-a6c9-a3c0648a0702.png)

**Desktop (please complete the following information):**
Win 10 V21H1 Build 19043.1165

**Comment by Ronsss** [comment]
Oops, that screen cap was not quite what I thought, here is a better one
![image](https://user-images.githubusercontent.com/33688532/130501841-bef86c75-f96e-42e6-8294-0b7a1f30cf64.png)

**Comment by rcmaehl** [comment]
I'm thinking it somehow got resized but it shouldn't have done so. I'll look into this.

**Comment by rcmaehl** [comment]
This should hopefully be patched per 4745d49. Please try it out when you have a chance

**Comment by micwoj92** [comment]
@Ronsss Could you try dev build from https://nightly.link/rcmaehl/WhyNotWin11/workflows/wnw11/main/WNW11.zip ? and check if it works?

**Comment by JohnLGalt** [comment]
> 
> 
> @Ronsss Could you try dev build from https://nightly.link/rcmaehl/WhyNotWin11/workflows/wnw11/main/WNW11.zip ? and check if it works?

I'm also running dual monitors of different resolutions, 1 is 4K (3840*2160) and the other is 1080p (1920*1080).  The 4K uses 150% DPI Scaling, the 1080 uses 100%.

Tested this on my machine, and it did not go wonky when switching from large to small and back again.

I tried to get it to open on the smaller monitor, but the app always opens to my primary monitor, the 4K one.
